examples/cms/src/main.tsx

```diff
@@ -0,0 +1,14 @@
+import "framer-plugin/framer.css"
+
+import React from "react"
+import ReactDOM from "react-dom/client"
+import { App } from "./App.tsx"
+
+const root = document.getElementById("root")
+if (!root) throw new Error("Root element not found")
+
+ReactDOM.createRoot(root).render(
+    <React.StrictMode>
+        <App />
+    </React.StrictMode>
+)
```

Comment:
I wonder if this would be a better place to add the logic for syncing an exists collection. Because then React is clearly completely skipped when nothing needs to be rendered. Wdyt?

Reply:
Nice one, will do!